spec/requests/api/v1/agents_spec.rb

```diff
@@ -0,0 +1,49 @@
+RSpec.describe "Agents API" do
+  let!(:oauth_token) { create(:access_token, resource_owner_id: agent.id, application:) }
+  let(:headers) do
+    { "Content-Type": "application/json", Authorization: "Bearer #{oauth_token.plaintext_token}" }
+  end
+  let(:application) { create(:oauth_application) }
+  let(:agent) { create(:agent, admin_role_in_organisations: [organisation]) }
+  let(:organisation) { create(:organisation) }
+
+  describe "#create" do
+    let(:params) do
+      {
+        email: "francis@factice.fr",
+        organisation_ids: [organisation.id],
+        access_level: "basic",
+      }
+    end
+
+    it "invites the agent into the organisation" do
+      post "/api/v1/agents", headers:, params:, as: :json
```

Comment:
Avec un `change`, on clarifie la spec, car on voit direct qu'il y a création, alors que sans, j'étais pas sûr de savoir si l'agent était simplement invité ou si il était aussi créé. 

```suggestion
      expect do
        post "/api/v1/agents", headers:, params:, as: :json
      end.to change(Agent, :count).by(1)
```

J'ai l'impression de suggérer ça assez souvent, il faut dire que j'adore ce matcher ! Je te laisse me dire si tu préfères ne pas utiliser `change` ou si c'est juste qu'on a pas les mêmes réflexes. :kissing_cat:

Reply:
hum je crois qu'il y a une question d'habitude que j'ai pas encore acquise. Je crois que c'est l'ajout d'un `expect do end` qui me donne l'impression que ça fait des spec qui expliquent moins directement ce qui se passe, mais clairement c'est subjectif. Bref, je suis chaud pour m'habituer à la joie de cette nouvelle manière de faire !